.travis.yml

```diff
@@ -1,15 +1,18 @@
 language: python
 python:
-  - "2.6"
+  # - "2.6"
   - "2.7"
   #- "3.1"  # cram does not support Python <3.2.4 anymore
   #- "3.2"
-  #- "3.3"   # but now it does support Python 3.3, according to release note 0.6
+  - "3.3"   # but now it does support Python 3.3, according to release note 0.6
+  - "3.4"
+  - "3.5"
+  - "3.6"
 branches:
   except:
     - piptools-ignore-patch
 install:
-  - "pip install cram --use-mirrors"
-  - "pip install . --use-mirrors"
+  - "pip install cram"
+  - "pip install ."
```

Comment:
OK, I've created the pull request. https://github.com/jgonggrijp/pip-review/pull/35

I've had multiple tiny pull requests be rejected in the past because the contributors don't want to have to merge or rebase the changes, so that's why I did it this way.